src/sfctl/helps/chaos_schedule.py

```diff
@@ -0,0 +1,54 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -----------------------------------------------------------------------------
+
+"""Help documentation for Service Fabric chaos-schedule commands"""
+
+from knack.help_files import helps
+
+helps['chaos-schedule set'] = """
+    type: command
+    short-summary: Set the Chaos Schedule to be used by Chaos.
+    long-summary: Set the Chaos Schedule currently in use by Chaos. Chaos will
+        automatically schedule runs based on the Chaos Schedule.
+        The version in the provided input schedule must match the version of
+        the Chaos Schedule on the server.
+        If the version provided does not match the version on the server, the
+        Chaos Schedule is not updated.
+        If the version provided matches the version on the server, then the
+        Chaos Schedule is updated and the version of the Chaos Schedule on the
+        server is incremented up by one and wraps back to 0 after 2,147,483,647.
+
```

Comment:
I would double check the formatting looks good here, sometimes linebreaks don't render correctly in resized windows.

Reply:
Since updating from master the output is no longer correct.

Reply:
Output looks like this

Command
    sfctl chaos schedule set: Set the Chaos Schedule to be used by Chaos.
        Set the Chaos Schedule currently in use by Chaos. Chaos will automatically schedule runs
        based on the Chaos Schedule. The version in the provided input schedule must match the
        version of the Chaos Schedule on the server. If the version provided does not match the
        version on the server, the Chaos Schedule is not updated. If the version provided matches
        the version on the server, then the Chaos Schedule is updated and the version of the Chaos
        Schedule on the server is incremented up by one and wraps back to 0 after 2,147,483,647.

        If Chaos is running when this call is made, the call will fail.

        Example
        The following command sets a schedule (assuming the current schedule has version 0) that
        starts on 2016-01-01 and expires on 2038-01-01 that runs Chaos 24 hours of the day, 7 days a
        week.

        sfctl chaos-schedule set --version 0 --start-date-utc "2016-01-01T00:00:00.000Z" --expiry-
        date-utc "2038-01-01T00:00:00.000Z" --chaos-parameters-dictionary [{\"Key\":\"adhoc\",\"Valu
        e\":{\"MaxConcurrentFaults\":3,\"EnableMoveReplicaFaults\":true,\"ChaosTargetFilter\":{\"Nod
        eTypeInclusionList\":[\"N0010Ref\",\"N0020Ref\",\"N0030Ref\",\"N0040Ref\",\"N0050Ref\"]},\"M
        axClusterStabilizationTimeoutInSeconds\":60,\"WaitTimeBetweenIterationsInSeconds\":15,\"Wait
        TimeBetweenFaultsInSeconds\":30,\"TimeToRunInSeconds\":\"600\",\"Context\":{\"Map\":{\"test\
        ":\"value\"}},\"ClusterHealthPolicy\":{\"MaxPercentUnhealthyNodes\":0,\"ConsiderWarningAsErr
        or\":true,\"MaxPercentUnhealthyApplications\":0}}}] --jobs [{\"ChaosParameters\":\"adhoc\",\
        "Days\":{\"Sunday\":true,\"Monday\":true,\"Tuesday\":true,\"Wednesday\":true,\"Thursday\":tr
        ue,\"Friday\":true,\"Saturday\":true},\"Times\":[{\"StartTime\":{\"Hour\":0,\"Minute\":0},\"
        EndTime\":{\"Hour\":23,\"Minute\":59}}]}].

is this acceptable?

Reply:
It's up to you in the end.

If it was my command, I would try to expand the `--chaos-parameters-dictionary` into something that's easier for humans to input. Including everything as a json string becomes very cumbersome for customers. Take a look at how [az vm create](https://docs.microsoft.com/en-us/cli/azure/vm?view=azure-cli-latest#az_vm_create) uses a bunch of parameters instead of a dictionary in JSON.

Reply:
the problem is that --chaos-parameters-dictionary is actually a list of 
{
  key: string,
  value: ChaosParameters
}
and ChaosParameters itself is really big. I can't really think of how to structure a command to take that in as multiple parameters. It wouldn't really make sense to have a parameter for each item in the list when you don't know how many items could be in the list.